service/worker/deployment/deployment_client.go

```diff
@@ -679,7 +680,20 @@ func (d *DeploymentClientImpl) updateWithStart(
 	}
 
 	policy := backoff.NewExponentialRetryPolicy(100 * time.Millisecond)
-	isRetryable := func(err error) bool { return errors.Is(err, errRetry) }
+	isRetryable := func(err error) bool {
+		if errors.Is(err, errRetry) {
+			return true
+		}
```

Comment:
Actually, could you do me a favor and run `common.IsServiceClientTransientError` on the err itself _as well_? I might have to patch MultiOps to return a vanilla serviceerror.Unavailable - but I can't do that if you only check for MultiOps itself.

Reply:
Discussed in-person; I forgot that there's already retries by the history client.